localstack/services/sns/provider.py

```diff
@@ -352,8 +352,18 @@ def create_platform_endpoint(
         return result
 
     def unsubscribe(self, context: RequestContext, subscription_arn: subscriptionARN) -> None:
-        call_moto(context)
-        store = self.get_store(account_id=context.account_id, region_name=context.region)
+        try:
+            parsed_arn = parse_arn(subscription_arn)
+        except InvalidArnException:
+            raise InvalidParameterException("Invalid parameter: SubscriptionId")
```

Comment:
can we have an AWS validated test for this exception? I don't think we have one. Something like `test_publish_wrong_arn_format`would be nice.